refactor/urls.js

```diff
@@ -1,16 +1,27 @@
-// Refactor this code
 
-function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+function buildURL(isDev = false, taskFlags = {}) {
```

Comment:
Why was the name of the function changed?

Reply:
I have reverted the change, Used previous function name.